packages/components/MenuButton/src/MenuButton.types.ts

```diff
@@ -0,0 +1,40 @@
+import { ContextualMenuItemProps, ContextualMenuProps } from '@fluentui-react-native/contextual-menu';
+import { IButtonProps } from '@fluentui-react-native/button';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+
+export const MenuButtonName = 'MenuButton';
+
+export interface MenuButtonContext {
+  showContextualMenu?: boolean;
+}
+
+export interface MenuButtonState {
+  context: MenuButtonContext;
+}
+
+export interface MenuButtonItemProps extends ContextualMenuItemProps {
+  submenu?: boolean;
+  submenuItems?: ContextualMenuItemProps[];
+}
+
+export interface MenuButtonProps extends IButtonProps {
+  menuItems?: MenuButtonItemProps[];
+  onItemClick?: (key: string) => void;
+  contextualMenu?: ContextualMenuProps;
+}
+
+export type MenuButtonSlotProps = {
+  root: MenuButtonProps;
+  button: IButtonProps;
+  contextualMenu: React.PropsWithRef<ContextualMenuProps>;
+  contextualMenuItems: Pick<MenuButtonProps, 'menuItems'>;
+  contextualMenuItem: MenuButtonItemProps;
+};
+
+export type MenuButtonRenderData = IRenderData<MenuButtonSlotProps, MenuButtonState>;
+
+export interface MenuButtonType {
+  props: MenuButtonProps;
+  slotProps: MenuButtonSlotProps;
+  state: MenuButtonState;
+}
```

Comment:
Does MenuButton require any tokens? What if someone wants to customize the Font/Color/Border of the MenuButton or the menu items?

Reply:
For now UI customization is not supported. @lenahong, do we need to support it for MenuButon?

Reply:
Ah that's a good point. We should discuss this actually. We can leave it out of this pr, but we should implement a way for clients to customize MenuButton to a certain extent.